README.md

```diff
@@ -158,12 +158,9 @@ class MyFavouriteVeryLongClassHolder : MyLongHolder<MyFavouriteVeryLongClass>(),
 3) Абстрактные методы
 4) Переопределенные методы родительского класса (желательно в порядке их следования в родительском классе)
 5) Реализации методов интерфейсов (желательно в порядке добавления интерфейсов в класс и следования методов в каждом интерфейсе)
-6) public методы
-7) internal методы
-8) protected методы
-9) private методы
-10) inner классы
-11) companion object
+6) Методы класса (в логическом порядке. Например, метод располагается после того, в котором впервые упомянут). 
```

Comment:
Хорошо написано в Kotlin Coding Convention, мб прямо цитатой вставить?
> Do not sort the method declarations alphabetically or by visibility, and do not separate regular methods from extension methods. Instead, put related stuff together, so that someone reading the class from top to bottom can follow the logic of what's happening. Choose an order (either higher-level stuff first, or vice versa) and stick to it.

Формально сортировка от открытого к закрытому тоже является логическим порядком, т.к. какую-то логику под собой подразумевает

Reply:
Может после списка типа "про порядок методов есть вот это:" и эта цитата?

Reply:
Угу, но просто сейчас это выглядит как переопределение правила из кодстайла Котлина, поэтому хочется подчеркнуть что мы ему следуе

Reply:
Покрутил и всё же не хочется цитату на английском сюда вставлять. В принипе там то же самое, что и "Например, метод располагается после того, в котором впервые упомянут" написано. Давай сделаем ссылку на это место лучше? 

Вот так: 
Методы класса ([в логическом порядке](https://kotlinlang.org/docs/coding-conventions.html#class-layout). Например, метод располагается после того, в котором впервые упомянут). 

Это будет ссылаться на гайд показывая, что в этом плане мы следуем ему. Что думаешь?

Reply:
Ага, ок

Reply:
Готово